packages/orbit-themer/.flowconfig

```diff
@@ -0,0 +1,6 @@
+[ignore]
+<PROJECT_ROOT>/node_modules/resolve/test/resolver/malformed_package_json/package.json
```

Comment:
Remove that `.flowconfig` and let's rely on the root config, add there that ignore rule

Reply:
We actually already have this rule there `.*/node_modules/rechoir/node_modules/resolve/test/resolver/malformed_package_json/.*` so it should be enough.